module/x/gravity/types/batch.go

```diff
@@ -51,9 +51,9 @@ func (b BatchTx) GetCheckpoint(gravityIDstring string) ([]byte, error) {
 		txAmounts,
 		txDestinations,
 		txFees,
-		big.NewInt(int64(b.BatchNonce)),
```

Comment:
We have a lot of nonces in this program. I think that prefixing their names, while it may be more verbose, is worth it for clarity.